### PR TITLE
CameraBrick: QActionGroup for an exclusive selection on image scale menu

### PR DESCRIPTION
**Pull request by natxo14**
Add QActionGroup to QtImport
Use it for a mutual exclusive menu when selecting scale on the right-click menu ( now, user can select several checkboxes at the same time )

### CODE REVIEW

**Comment by IvarsKarpics** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ gui/bricks/CameraBrick.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/mxcube/mxcube/pullRequest?prid=4976931&bid=16439464)

File: gui/bricks/CameraBrick.py [hunk 1]
```diff
@@ -236,6 +236,7 @@ def property_changed(self, property_name, old_value, new_value):
                 self.connect(HWR.beamline.microscope, "infoMsg", self.set_info_msg)
                 self.graphics_view = HWR.beamline.microscope.get_graphics_view()
                 # self.graphics_camera_frame = HWR.beamline.microscope.get_camera_frame()
+                self.graphics_view.resize(700,700)
```
Comment (IvarsKarpics) [inline gui/bricks/CameraBrick.py]: It would be good to define the size via property instead of hard coded values.

**Review by IvarsKarpics — CHANGES_REQUESTED** [review]

File: gui/bricks/CameraBrick.py (hunk 1 above)
Comment (natxo14) [reply gui/bricks/CameraBrick.py]: Completely agree.
I was making some tests on widget's size and how it changed the global rendering of the app; I forgot to change that. I'll fix it soon.
Thanks!

Comment (natxo14) [reply gui/bricks/CameraBrick.py]: Finally I removed the 'resize()' call: 
on my experience with Qt, is better to rely on Qt's layout management unless we're looking for a very precise sizing of widget's inserted in layouts.

**Comment by IvarsKarpics** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ gui/bricks/CameraBrick.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/mxcube/mxcube/pullRequest?prid=4976931&bid=16439464)

**Pull request merged**